### DISCUSSION TITLE
How to use bubbletea and gum in existing cli which was built using cobra cli framework?

### DISCUSSION THREAD

**Discussion by devopstoday11** [charmbracelet/bubbletea#940] [Q&A]
Hi,
Are there any guidelines to use bubbletea TUI framework and gum in our existing cli (built using cobra cli framework, the same framework used to build Kubernetes cli kubectl)?

**Comment by KevM** [comment]
We use pflags to get arguments from the user and then use those arguments to create a bubble tea model that we pass to `tea.NewProgram(model)`
